testing/types/expected_keepers.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+import (
+	"context"
+
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+// StakingKeeper defines the expected staking keeper interface used in the
+// IBC testing package
+type StakingKeeper interface {
```

Comment:
Is this needed now that we have the eureka changes?

Reply:
Nope! Good catch, removing now 🙏